src/LinkDotNet.Blog.Domain/BlogPost.cs

```diff
@@ -35,6 +38,35 @@ private BlogPost()
 
     public string TagsAsString => Tags is null ? string.Empty : string.Join(", ", Tags);
 
+    public string SearchEngineFriendlyUrl => GenerateSearchEngineFriendlyUrl();
+
+    private string GenerateSearchEngineFriendlyUrl()
+    {
+        // Remove all accents and make the string lower case.
+        if (string.IsNullOrWhiteSpace(Title))
+            return Title;
+
+        Title = Title.Normalize(NormalizationForm.FormD);
+        char[] chars = Title
+            .Where(c => CharUnicodeInfo.GetUnicodeCategory(c)
+            != UnicodeCategory.NonSpacingMark).ToArray();
+
+        Title = new string(chars).Normalize(NormalizationForm.FormC);
+
+        string SearchEngineFriendlyTitle = Title.ToLower(CultureInfo.CurrentCulture);
+
+        // Remove all special characters from the string.  
+        SearchEngineFriendlyTitle = Regex.Replace(SearchEngineFriendlyTitle, @"[^A-Za-z0-9\s-]", "");
```

Comment:
The Regex can be moved into private partial RegEx - with source generators

Reply:
Well, I've never done this before let me try. I may ask you for help.

Reply:
Funny I guess I found the answer on your blog 
https://steven-giesel.com/blogPost/667db4f5-d34a-4625-bc3d-aa47187c591a

Reply:
Haha very nice! I like that

Reply:
I did my best but I'm not sure that I did it correctly.